Testing/functional/tests/cypress/integration/e2e/dependent/dashboard.js

```diff
@@ -77,6 +80,10 @@ describe("dependents - dashboard", () => {
         validateDatasetCard("ClinicalDocument");
     });
 
+    it("Validate dashboard clinical documents tab click to timeline", () => {
```

Comment:
Test name should reference hospital visits rather than clinical documents.

Reply:
Fixed and pushed